pyproject.toml

```diff
@@ -38,6 +38,8 @@ sentry-sdk = "^1.0.0"
 pysilverpop = "^0.2.6"
 lxml = "^4.6.3"
 prometheus-client = "^0.10.1"
+structlog = "^21.1.0"
+colorama = "^0.4.4"
```

Comment:
_this enables some colorization for output, I'm interested how this plays with the structlog for console outputting._